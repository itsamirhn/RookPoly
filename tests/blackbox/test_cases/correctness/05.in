3
100
000
011
