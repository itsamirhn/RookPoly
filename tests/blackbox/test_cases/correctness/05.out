2X^3 + 8X^2 + 6X + 1
